src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -36,10 +44,11 @@ const PageLayoutWrapper = ({
 
   const scriptSwitchId = pathOr('', ['scriptSwitchId'], pageData);
   const renderScriptSwitch = pathOr(true, ['renderScriptSwitch'], pageData);
-  const isErrorPage = [404, 500].includes(status);
-  const pageType = isErrorPage
-    ? 'WS-ERROR-PAGE'
-    : path<string>(['metadata', 'type'], pageData);
+
+  const isErrorPage = ![200].includes(status) || !status;
+  const cpsPageType = pageData?.metadata?.type;
+  const otherPageType = pageData?.pageType;
```

Comment:
`pageType` is available on the RequestContext (will need to be imported). We should use it as the fallback page type, in case there is no page type on the pageData object - this is a bit cleaner than this solution

```suggestion
  const { pageType as fallbackPageType } = useContext(RequestContext);
```